global.json

```diff
@@ -0,0 +1,11 @@
+{
+  "sdk": {
+    "version": "2.1.300-preview3-008414"
```

Comment:
To avoid a patch, this should be `2.1.300-preview2-008533` to match the preview2 SDK we use in source-build.

Reply:
Isn't `release/2.1.3xx-src` used for source build currently?

Reply:
Actually, soon we'll have a `global.json` updater that would replace a patch, so it's probably fine to patch/branch for this change temporarily. LGTM as is.

Reply:
I'm saying that we should merge `release/2.1.3xx-src` into `release/2.1.3xx` if it is possible to do so now. Then delete `release/2.1.3xx-src`.

Reply:
Isn't that basically what this PR is doing? How does that relate to the SDK version?

Reply:
I guess it is, but there might have been more changes in the -src branch since this PR was created. I think it would be better to create a new PR that merges the branch.